common-service-core/src/main/java/org/zowe/apiml/auth/Authentication.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * This program and the accompanying materials are made available under the terms of the
+ * Eclipse Public License v2.0 which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-v20.html
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Copyright Contributors to the Zowe Project.
+ */
+package org.zowe.apiml.auth;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.*;
+
+/**
+ * Information about expected authentication scheme and APPLID for PassTickets generation.
+ */
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class Authentication {
+
+    private AuthenticationScheme scheme;
+    private String applid;
+
+    @JsonIgnore
```

Comment:
This value represents the state of metadata, not the actual SSO support. We don't want to display it in JSON. Getter would allow making a mistake. Method `supportsSso()` should be used instead.